src/explore-education-statistics-admin/src/services/releaseDataFileService.ts

```diff
@@ -38,6 +38,7 @@ export interface DataFile {
   created?: string;
   isDeleting?: boolean;
   isCancelling?: boolean;
+  isLinkedToApiDataSet?: boolean;
```

Comment:
As per earlier comments - this has been replaced with the following:

- `publicApiDataSetId` (string) - can be used for linking
- `publicApiDataSetVersion` (string) - can't be used for linking, but added for completeness and potential future use-cases / debugging